provisioning/image_customization.go

```diff
@@ -51,13 +57,41 @@ func setIronicBaseUrl(name string, info *ProvisioningInfo) corev1.EnvVar {
 	}
 }
 
+func setImagePublishUrl(imageCustomizationService string, imagesBindAddress string) corev1.EnvVar {
+	ips, err := net.LookupIP(imageCustomizationService)
```

Comment:
We can just use the internal DNS name I think?

Reply:
Done.